docs/SUMMARY.md

```diff
@@ -0,0 +1,12 @@
+<!-- markdownlint-disable -->
```

Comment:
Why disabling here?

Reply:
Otherwise, these errors will come


- First-line in a file should be a top-level heading
- indent Unordered list indentation 

This file is for the navbar. We need to write it that way. We don't need any heading, and indentation in that fashion is required to make the site look what it looks like.